oidc_cli/token.go

```diff
@@ -0,0 +1,79 @@
+package client
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	timeSkew = 5 * time.Minute
+
+	tokenVersion = 0
+)
+
+// Claims represent the oidc token claims
```

Comment:
TODO: this needs more thought to be useful as a shared library.
I'll open a follow-up pr later